Travis is failing on pypy2.5.0

Looks like something in the new dictionary implementation is failing on travis: 

```
======================================================================
FAIL: TestFileTiff.test___str__
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python-pillow/Pillow/Tests/test_file_tiff.py", line 181, in test___str__
    '{256: (55,), 257: (43,), 258: (8, 8, 8, 8), 259: (1,), '
AssertionError: '{259: (1,), 284: (1,), 262: (2,), 256: (55,), 257: (43,), 258: (8, 8, 8, 8), 338: (1,), 282: ((720000, 10000),), 283: ((720000, 10000),), 296: (2,), 273: (8,), 277: (4,), 279: (9460,)}' != '{256: (55,), 257: (43,), 258: (8, 8, 8, 8), 259: (1,), 262: (2,), 296: (2,), 273: (8,), 338: (1,), 277: (4,), 279: (9460,), 282: ((720000, 10000),), 283: ((720000, 10000),), 284: (1,)}'
```

Looks like a dictionary ordering issue to me.  I think that this is a test issue, not a bug in Pillow.

--- a/Tests/test_file_tiff.py
+++ b/Tests/test_file_tiff.py
@@ -176,12 +176,23 @@ class TestFileTiff(PillowTestCase):
 
         # Assert
         self.assertIsInstance(ret, str)
+
+    def test_as_dict(self):
+        # Arrange
+        filename = "Tests/images/pil136.tiff"
+        im = Image.open(filename)
+
+        # Act
+        ret = im.ifd.as_dict()
+
+        # Assert
+        self.assertIsInstance(ret, dict)
+
         self.assertEqual(
-            ret,
-            '{256: (55,), 257: (43,), 258: (8, 8, 8, 8), 259: (1,), '
-            '262: (2,), 296: (2,), 273: (8,), 338: (1,), 277: (4,), '
-            '279: (9460,), 282: ((720000, 10000),), '
-            '283: ((720000, 10000),), 284: (1,)}')
+            ret, {256: (55,), 257: (43,), 258: (8, 8, 8, 8), 259: (1,),
+                  262: (2,), 296: (2,), 273: (8,), 338: (1,), 277: (4,),
+                  279: (9460,), 282: ((720000, 10000),),
+                  283: ((720000, 10000),), 284: (1,)})
 
     def test__delitem__(self):
         # Arrange

